nncf/quantization/algorithms/bias_correction/onnx_backend.py

```diff
@@ -77,12 +76,12 @@ def output_insertion_command(nncf_graph: NNCFGraph, target_point: ONNXTargetPoin
 
     @staticmethod
     def mean_statistic_collector(
-        reduction_shape: ReductionShape,
+        channel_axis: int,
         inplace: bool,
         num_samples: Optional[int] = None,
         window_size: Optional[int] = None,
     ) -> ONNXMeanStatisticCollector:
-        return ONNXMeanStatisticCollector(reduction_shape, num_samples, window_size)
+        return ONNXMeanStatisticCollector(channel_axis, num_samples, window_size)
```

Comment:
As far as understand `ONNXMeanStatisticCollector` is inhered from `OfflineTensorStatisticCollector` which is waiting `reduction_shape` https://github.com/daniil-lyakhov/nncf/blob/55118d5121f0188100c7539c25a50768a47686f3/nncf/common/tensor_statistics/collectors.py#L103C15-L103C30

Please, check that this changes is consistent.